app/controllers/administrator_controller.rb

```diff
@@ -0,0 +1,43 @@
+class AdministratorController < ApplicationController
+
+  def index
+  	authorize! :index, :administrator
+     @users = User.paginate(page: params[:page])
```

Comment:
Use `User.all` for now rather than paginate, since we don't have page navigation developed yet